### DISCUSSION TITLE
Custom message for manual flows

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#16155] [Feature Requests]
In this particular case, there isn't really an extension type that can be used to send a notification item like that when a flow completes. In this case, the best way forwards is to create a new feature request to allow the user to configure a custom message based on the flow data when the manual flow completes :)

_Originally posted by @rijkvanzanten in https://github.com/directus/directus/discussions/15193#discussioncomment-3512513_

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
